http/swagger.yaml

```diff
@@ -43,6 +43,8 @@ paths:
           description: Message is accepted for processing.
         400:
           description: Message discarded due to its malformed content.
+        401:
```

Comment:
This is wrong and misleading. 401 should be thrown exclusively on missing credentials (token). Permissions issues should be taken care via 403. Please review the other API docs as well.